### PR TITLE
wireframe for basic components

### PR DESCRIPTION
**Pull request by rkzel**
With responsiveness, but without much styling.

### CODE REVIEW

**Comment by Schwartz10** [comment]
Hey @rkzel - a couple things:

1. Did you commit the `EducationPanel` and `WorkHistoryPanel`? I can't get the code to run because I don't have these files.
2. Please remove all the console logs before committing 
3. please remove the changes to the package.json - we don't need to require Orbit and once 3box releases their next build that bug you were experiencing should be fixed

File: app/src/components/OrganizationPanel.js [hunk 1]
```diff
@@ -1,35 +1,20 @@
 import React from 'react'
 import styled from 'styled-components'
 import { Card, Text } from '@aragon/ui'
+import CardWrapper from './CardWrapper'
+
+const addMore = () => console.log('add more orgs')
```
Comment (Schwartz10) [inline app/src/components/OrganizationPanel.js]: this console.log is fine to keep for now

File: app/src/components/Profile.js [hunk 2]
```diff
@@ -1,17 +1,40 @@
...
+import WorkHistoryPanel from './WorkHistoryPanel'
 
 const Profile = ({ ethereumAddress }) => {
   return (
-    <div>
-      <PanelSeparator>
-        <InformationPanel ethereumAddress={ethereumAddress} />
-        <OrganizationPanel />
-      </PanelSeparator>
+    <div style={{ width: '100%' }}>
```
Comment (Schwartz10) [inline app/src/components/Profile.js]: Do you think we should stick to styled-components only? Or are inline styles ok too? What's a good rule of thumb for this?

File: app/src/components/informationPanel/InformationCard.js [hunk 3]
```diff
@@ -17,55 +18,51 @@ const InformationCard = ({ ethereumAddress }) => {
     return boxes[ethereumAddress].publicProfile[field] || ''
   }
 
+  console.log('what is there: ', boxes[ethereumAddress])
```
Comment (Schwartz10) [inline app/src/components/informationPanel/InformationCard.js]: remove plz

File: app/src/components/Profile.js (hunk 2 above)
Comment (rkzel) [reply app/src/components/Profile.js]: Talked on Keybase

File: app/src/components/informationPanel/InformationCard.js (hunk 3 above)
Comment (rkzel) [reply app/src/components/informationPanel/InformationCard.js]: Removed.

File: app/src/components/OrganizationPanel.js (hunk 1 above)
Comment (rkzel) [reply app/src/components/OrganizationPanel.js]: Just for now :)

**Pull request merged**